### PR TITLE
[RAGEMP] 0.3.7 to 1.1+ dotnet image

### PR DESCRIPTION
**Pull request by DeadSurfer**
1. Changed server version from old 0.3.7 to last 1.1
2. Changed docker image to support dotnet
3. Changed maxPlayers var, ragemp dont have a limit for players

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

### CODE REVIEW

File: gta/ragemp/egg-rage--m-p.json [hunk 1]
```diff
@@ -39,7 +39,7 @@
             "default_value": "50",
             "user_viewable": 1,
             "user_editable": 0,
-            "rules": "required|integer|between:1,100"
+            "rules": "required|integer|between:1,10000"
```
[the remark this answers is not in the document]
Comment (DeadSurfer) [reply gta/ragemp/egg-rage--m-p.json:42]: no limits, ragemp have much servers with 1k players

[the remark this answers is not in the document]
Comment (DeadSurfer) [reply gta/ragemp/egg-rage--m-p.json:42]: uh this is rules wtf, my mistake ))))

File: gta/ragemp/egg-rage--m-p.json [hunk 2]
```diff
@@ -7,17 +7,17 @@
...
+            "script": "apt update\r\napt -y install curl tar git wget\r\n\r\ncd \/mnt\/server\r\n\r\necho \"Downloading rage.mp\"\r\ncurl -sSL -o linux_x64.tar.gz https:\/\/cdn.rage.mp\/updater\/10_mNwuchuQ4ktWbR8d2N5jUzRt\/server-files\/linux_x64.tar.gz\r\n\r\ntar -xzvf linux_x64.tar.gz --strip 1 -C \/mnt\/server\r\n\r\nrm linux_x64.tar.gz\r\n\r\nchmod +x .\/ragemp-server\r\n\r\nif [ -e conf.json ]; then\r\n    echo \"server config file exists\"\r\nelse\r\n    echo \"Downloading default rage.mp config\"\r\n    curl https:\/\/raw.githubusercontent.com\/parkervcp\/eggs\/master\/gta\/ragemp\/conf.json >> conf.json\r\nfi\r\n\r\necho \"install complete\"\r\n\r\nexit 0",
```
Comment (parkervcp) [inline gta/ragemp/egg-rage--m-p.json:20]: Per the wiki the link is different

https://wiki.rage.mp/index.php?title=Getting_Started_with_Server#Debian

File: gta/ragemp/egg-rage--m-p.json (hunk 1 above)
[the remark this answers is not in the document]
Comment (TASelwyn) [reply gta/ragemp/egg-rage--m-p.json:42]: Also, if you are changing the player limit to 10k, you should change the variable description to reflect that.

[the remark this answers is not in the document]
Comment (DeadSurfer) [reply gta/ragemp/egg-rage--m-p.json:42]: > Why 10000. I'm absolutely certain 100 is too high...

maybe u ll accept pull request ?)

**Comment by waffleuncle** [comment]
The server seems to be working. 
But it still displays Starting even after successfully starting.

In the console I can observe the following line that indicates that the server is ready for the connection.

[DONE] The server is ready to accept connections.

**Pull request merged**